python_package/examples/tests/pieeg_client.py

```diff
@@ -0,0 +1,83 @@
+import logging
+import socket
```

Comment:
we maintain only minimalistic examples, its ok to create smth like brainflow_get_data.py for pieeg only, but stuff like sending data, etc is outside from the scope of these examples, so please remove

Reply:
Removed pieeg_client.py as it is outside the scope of minimalistic examples maintained by the project.